MANIFEST.in

```diff
@@ -3,3 +3,6 @@ include LICENSE
 include requires-*.txt
 include dash/favicon.ico
 include dash/extract-meta.js
+include dash/deps/*.js
+recursive-include dash/dash-renderer *
```

Comment:
This will include all the source files, won't it? I don't think we want to do that. The old dash-renderer `MANIFEST.in` is just:
```
include package.json
include digest.json
include dash_renderer/*.js
include dash_renderer/*.map
```
In the new structure, `dash_renderer/*.js` is replaced by `dash/deps/*.js` which you have above. I'm not aware of any value that `package.json` or `digest.json` have to end users so we should be able to ignore those...

Then there's the question of the `*.map` files: you're not currently including them, and in fact in the past in response to a user request to reduce the package size https://github.com/plotly/dash/issues/810 we at one point removed them https://github.com/plotly/dash/pull/910 but then for some reason that I can't find any record of we immediately reverted that https://github.com/plotly/dash/commit/369ace90b4af72d8ace1daaae9479b1129a462ee before making a release. @Marc-Andre-Rivet do you have any memory of that? I don't see a PR or any discussion, just a revert commit

Anyway, is there anything else we really do want from the dash-renderer directory, or can we drop this line (as well as the line below that only exists in reference to this one)?

Reply:
Updated in afc363cc. We can safely drop the other lines

I'm noticing that `.map` files aren't being generated by the webpack for souce-mapping. Is this because we have `inline-source-map`?

Reply:
Ah hmm, actually this changed in #1576 in a way that I hadn't recognized - @Marc-Andre-Rivet please confirm I'm understanding this correctly: the ONLY case we're generating a sourcemap now is when you run `webpack-serve` (which is what you get with `npm start`). The other build commands now seem to be all the same (as far as the bundle itself is concerned) and create no sourcemap. `npm run build:js` and `build:dev` are now literally the same command.

Unfortunately I can't actually run `npm start` ATM, and I don't think it does what we want anyway - really I'd like a command that outputs the same files we currently create, but stays open watching the source files for changes. @Marc-Andre-Rivet didn't we have that in the renderer at some point? Am I missing something about the updated tooling?